microcosm_pubsub/handlers.py

```diff
@@ -82,7 +82,7 @@ def on_skip(self, message, uri, reason):
         )
 
     def on_handle(self, message, uri, resource):
-        self.logger.info(
+        self.logger.debug(
```

Comment:
I'm basically moving this log a layer up so we can track elapsed time of the whole handler. I'm leaving the log because having `on_handle` in the `URIHandler` api is probably useful for other things.